urbs.py

```diff
@@ -1537,7 +1556,7 @@ def get_entity(instance, name):
         instance: a Pyomo ConcreteModel instance
         name: name of a Set, Param, Var, Constraint or Objective
 
-    Returns:
+    Returns
```

Comment:
Minor stylistic critique: usually such edits should be done in focused, dedicated commits. In addition to that: the whole file currently does use both `Returns:` with and without a colon, so removing it only here is inconsequential at best. I would leave it alone for now.